discourse_automation/llm_triage.rb

```diff
@@ -52,6 +52,8 @@
       canned_reply = fields.dig("canned_reply", "value")
       canned_reply_user = fields.dig("canned_reply_user", "value")
 
+      next if post.user.username == canned_reply_user
```

Comment:
Micro optimisation here but we can move this up the sequence in the block to return earlier instead of doing alot of work to gather the variables which may be thrown away.